main.cpp

```diff
@@ -70,5 +70,16 @@ int main(int argc, char *argv[])
 //    testFile.close();
     //-------------------------------------------------------------------
 
+    // load the ROM file from argv if exists
+    if (argc > 1)
+    {
+        QString filePath(argv[1]);
+        ROMUtils::FormatPathSeperators(filePath);
+        if(QFile::exists(filePath) && filePath.endsWith(".gba", Qt::CaseInsensitive))
```

Comment:
If a parameter is given to load the ROM but this inner if statement fails, it should print some warning to console that File path was invalid or file was not a .gba file

Reply:
solution: pop up critical dialog to show error.